.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -1,7 +1,7 @@
 name: 🐞 Bug
```

```suggestion
name: Bug
```

We'll need to remove emojis :(

---

Leaving since it's allowed. 🥳